scripts/linters/test_files/invalid_duplicate_jobs_one_off.py

```diff
@@ -29,7 +29,7 @@
     models.NAMES.base_model, models.NAMES.collection])
 
 
-class CollectionMigrationOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+class IndexAllActivitiesJobManager(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
is this an expected change? [if yes, can we the collection term from the dummy docstrings?]

Reply:
Done.